src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -51,26 +57,48 @@ const AssetList: FunctionComponent<AssetListProps> = ({
   className,
   featureTypes,
 }) => {
-  const { id, type, isReported } = selection
+  const { id, type, isReported, isChecked } = selection
   const { description, icon }: Partial<FeatureType> =
     featureTypes?.find(({ typeValue }) => typeValue === type) ?? {}
 
   const label = [description, id].filter(Boolean).join(' - ')
 
   if (!id) return null
 
+  const reportedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'reported'
+  ) as ReportedFeatureType
+
+  const checkedFeatureType = featureTypes.find(
+    ({ typeValue }) => typeValue === 'checked'
+  ) as CheckedFeatureType
+
   return (
     <IconList data-testid="assetList" className={className}>
       <IconListItem
         key={id}
-        id={isReported ? `assetListItem-${id}-reported` : `assetListItem-${id}`}
+        id={
+          isReported
+            ? `assetListItem-${id}-reported`
+            : isChecked
+            ? `assetListItem-${id}-checked`
+            : `assetListItem-${id}`
+        }
         iconUrl={icon?.iconUrl}
         isReported={isReported}
+        isChecked={isChecked}
       >
         <ItemWrapper>
           <StyledLabel>
             {label}
-            {isReported && <StyledDiv>Is gemeld</StyledDiv>}
+            {isReported && (
+              <StyledDiv isReported>
```

Comment:
Maybe use a more descriptive name instead of `StyledDiv`?

Reply:
✅